src/drivers/intel/hda/hda-dma.c

```diff
@@ -680,6 +682,7 @@ static int hda_dma_set_config(struct dma_chan_data *channel,
 	channel->desc_count = config->elem_array.count;
 	channel->is_scheduling_source = config->is_scheduling_source;
 	channel->period = config->period;
+	channel->two_step_stop = config->two_step_stop;
```

Comment:
if we had a link from channel to config we could avoid duplication of is_scheduling_source and two_step_stop, no?